doc/meson.build

```diff
@@ -25,7 +25,7 @@ if want_man or want_html
     # docbooks = [ ['file1', 'manvolnum-from-file1.xml', 'file1.xml'],
     #              ['file2', 'manvolnum-from-file2.xml', 'file2.xml'], ... ]
     docbooks = []
-    rr = run_command(docbklst)
+    rr = run_command(docbklst, check: false)
```

Comment:
Is there a valid reason for this to ever fail?

Generically speaking, the reason Meson is advocating to explicitly set a check: value (and considering changing the default in the future) is because `check: false` is a bad implicit default -- you generally want to use `check: true` since erroring commands shouldn't be treated the same as "output is an empty string".

Reply:
Thanks for the explanation @eli-schwartz 
I need to revisit this code when I get a chance.